stock_picking_invoice_link/wizards/account_move_reversal.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Tecnativa - Carlos Roca
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import models
+
+
+class AccountMoveReversal(models.TransientModel):
+    _inherit = "account.move.reversal"
+
+    def reverse_moves(self):
```

Comment:
Hello @lethuthao2403, please manage the new param `is_modify=False` in the method `reverse_moves`. Please see this [commit](https://github.com/odoo/odoo/commit/4d3ac4cbd87fa) for more details

Reply:
hi @dzungtran89 the PR has been updated! Thanks.